google/ggrc-core
# GGRC-6272 Mapped objects don't have titles on Cycle Task's Edit/Create modal

# Issue description

Mapped objects don't have titles on Cycle Task's Edit/Create modal

# Steps to test the changes

1. Open Edit/Create modal for Cycle Task.
2. Click Map Objects.
3. Using mapper click "All" link to map all objects to Cycle Task.
**Actual result**: Several objects have an empty title.
**Expected result**: All objects should have title.

Time to load objects (id, type, title, viewLink) for 100 Access Groups: ~3.905 sec

# Solution description

Create new component mapping-list-loader to operate with list of mapped objects and to make query api request to get the required fields for new mapped objects.

# Sanity checklist


# PR Review checklist
